http/client/client.go

```diff
@@ -31,7 +31,7 @@ type Token string
 
 func (t Token) Set(req *http.Request) {
 	if string(t) != "" {
-		req.Header.Set("Authorization", fmt.Sprintf("Scope-Probe token=%s", t))
+		req.Header.Set("Authorization", fmt.Sprintf("Scoper-Probe token=%s", t))
```

Comment:
Oops

Reply:
Darn, I switched to Goland and it seems automatic refactorings are not woring as I expected. Will fix.